clang/include/clang/Serialization/ASTReader.h

```diff
@@ -1780,12 +1780,13 @@ class ASTReader
   /// Read the control block for the named AST file.
   ///
   /// \returns true if an error occurred, false otherwise.
-  static bool readASTFileControlBlock(StringRef Filename, FileManager &FileMgr,
-                                      const InMemoryModuleCache &ModuleCache,
-                                      const PCHContainerReader &PCHContainerRdr,
-                                      bool FindModuleFileExtensions,
-                                      ASTReaderListener &Listener,
-                                      bool ValidateDiagnosticOptions);
+  static bool readASTFileControlBlock(
+      StringRef Filename, FileManager &FileMgr,
+      const InMemoryModuleCache &ModuleCache,
+      const PCHContainerReader &PCHContainerRdr, bool FindModuleFileExtensions,
+      ASTReaderListener &Listener, bool ValidateDiagnosticOptions,
+      unsigned DefaultClientLoadCapabilities = ARR_ConfigurationMismatch |
```

Comment:
Shouldn't this be just `ClientLoadCapabilities`? Not sure what the meaning of "default" is here, especially if the caller actually explicitly specifies something than `ARR_ConfigurationMismatch | ARR_OutOfDate`.